common/src/main/java/com/LubieKakao1212/opencu/common/device/TrackerBase.java

```diff
@@ -137,12 +171,11 @@ private boolean drainEnergy(BlockEntityModularFrame.ModularFrameContext ctx, Tra
         if (used != energyToUse) {
             state.noEnergy = true;
             return false;
-        }
-        else {
+        } else {
```

Comment:
previous formatting behaves better when folding
```suggestion
        } 
        else {
```

Reply:
This one was not on purpose so no worries.